sig-aws/charter.md

```diff
@@ -0,0 +1,55 @@
+# SIG AWS Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG AWS is responsible for the creation and maintenance of subprojects (features/innovations) necessary to operate Kubernetes on AWS (see below). SIG AWS acts as a forum for Kubernetes on AWS users/developers to raise their feature requests and support issues with. SIG AWS in collaboration with SIG-Testing, SIG-Scalability and SIG-Docs is responsible for integration and maintenance of tests, scale-tests and documentation for all things Kubernetes on AWS.
+
+### In scope
+
+Link to SIG section in [sigs.yaml](https://github.com/kubernetes/community/blob/master/sigs.yaml) and SIG [subprojects](https://github.com/kubernetes/community/tree/master/sig-aws#subprojects)
```

Comment:
@justinsb - this means that all the listing of subprojects in sigs.yaml are within scope. This is the format from the charter template. I will improve the verbiage as it may confuse a new member.

Reply:
Personally I would nuke this explicit sentence entirely.  I've preferred charters that explicitly list scope without punting to other documents.

[SIG VMWare](https://github.com/kubernetes/community/blob/master/sig-vmware/charter.md) is a good example as another cloud-focused SIG.  Other good examples include [SIG Cluster Lifecycle](https://github.com/kubernetes/community/blob/master/sig-cluster-lifecycle/charter.md) and [SIG Instrumentation](https://github.com/kubernetes/community/blob/master/sig-instrumentation/charter.md)